.github/pull_request_template.md

```diff
@@ -0,0 +1,32 @@
+**CHANGE DESCRIPTION**
+---
+**Problem:**
+*Short explanation of the problem.*
+
+**Cause:**
+*Short explanation of the root cause of the issue if applicable.*
+
+**Solution:**
+*Short explanation of the solution we are providing with this PR.*
+
+**CHECKLIST**
+---
+**Jira**
+- [ ] Is the Jira ticket created and referenced properly?
+- [ ] Does the Jira ticket have the proper statuses for documentation (`Needs Doc`) and QA (`Needs QA`)?
+- [ ] Does the Jira ticket link to the proper milestone (Fix Version field)?
+
+**Tests**
```

Comment:
I think we already mentioned this somewhere, but do we need this section since it is already visible to us?

Reply:
For now it was decided to leave it (community contributions etc.), if it will not make sense later we will remove it.